### ISSUE TITLE
How to train model from scratch

### ISSUE THREAD

**Issue by MiaoDX** [scottworkman/deephorizon#4]
Hi!

The demo is charming! Do you have any plan to open-source the training code. There seems to be one `solver.proto` and `deploy.net` in the shared files, but the trained model is "train.net", is there any difference between deploy and train, and in my short experience of caffe usage, there should be some difference like the data and loss layer, as shown in [https://github.com/BVLC/caffe/wiki/Using-a-Trained-Network:-Deploy](https://github.com/BVLC/caffe/wiki/Using-a-Trained-Network:-Deploy).

Could you provide the train.net or give some hints?

**Comment by scottworkman** [comment]
Hello!

The "deploy.net" protocol buffer is virtually identical to the "train.net" version, minus the data loading layers and the loss layers. 

For example: https://github.com/BVLC/caffe/blob/master/models/bvlc_googlenet/train_val.prototxt

**Comment by MiaoDX** [comment]
Yeah, it is because some network will contains drop out and some other tricks, if they are just almost the same, I will try on the downloaded data and see if I can reproduce the result :)
